frontend/e2e/$lang+/_public/apply/adult-child/adult-child-flow.spec.ts

```diff
@@ -0,0 +1,415 @@
+  });
+
+  test('Should complete adult and child application', async ({ page }) => {
+    await test.step('Should navigate to type of application page', async () => {
+      await typeOfApplication(page);
+    });
+
+    await test.step('Should navigate to tax filing page', async () => {
+      await taxFiling(page);
+    });
+
+    await test.step('Shoud navigate to file your taxes page', async () => {
+      await fileTaxes(page);
+    });
+
+    // Applicant age is 65 or older, child is under 18
+    await test.step('Should navigate to date of birth page', async () => {
+      await expect(page).toHaveURL(/\/en\/apply\/[a-f0-9-]+\/adult-child\/date-of-birth/);
+      await expect(page.getByRole('heading', { level: 1, name: 'Age' })).toBeVisible();
+      await page.getByRole('button', { name: 'Continue' }).click();
+
+      // check empty fields
+      await hasError(page, 'Date of birth must include a year');
+      await hasError(page, 'Date of birth must include a month');
+      await hasError(page, 'Date of birth must include a day');
+      await hasError(page, 'Select wether or not all the children are under 18');
+
+      await fillOutDOB(page, '1958', '01', '01', 'Yes');
+      await page.getByRole('button', { name: 'Continue' }).click();
+    });
+
+    await test.step('Should navigate to applicant information page when applicant is 18-64', async () => {
+      await expect(page).toHaveURL(/\/en\/apply\/[a-f0-9-]+\/adult-child\/applicant-information/);
+      await expect(page.getByRole('heading', { level: 1, name: 'Applicant information' })).toBeVisible();
+
+      // Back to date of birth
+      await page.getByRole('link', { name: 'Back' }).click();
+      await expect(page).toHaveURL(/\/en\/apply\/[a-f0-9-]+\/adult-child\/date-of-birth/);
+      await expect(page.getByRole('heading', { level: 1, name: 'Age' })).toBeVisible();
+
+      await fillOutDOB(page, '1985', '02', '10', 'Yes');
+      await page.getByRole('button', { name: 'Continue' }).click();
+    });
+
+    await test.step('Should navigate to DTC page', async () => {
+      await expect(page).toHaveURL(/\/en\/apply\/[a-f0-9-]+\/adult-child\/disability-tax-credit/);
+      await expect(page.getByRole('heading', { level: 1, name: 'Disability tax credit' })).toBeVisible();
+      await page.getByRole('button', { name: 'Continue' }).click();
+
+      //check for empty fields
+      await hasError(page, 'Select whether or not you have a valid disability tax credit');
+
+      await page.getByRole('radio', { name: 'No' }).check();
+      await page.getByRole('button', { name: 'Continue' }).click();
+    });
+
+    await test.step('Should navigate to apply for your children page if applicant has no DTC, child is under 18', async () => {
+      await expect(page).toHaveURL(/\/en\/apply\/[a-f0-9-]+\/adult-child\/apply-children/);
+      await expect(page.getByRole('heading', { level: 1, name: 'Apply for your child(ren)' })).toBeVisible();
+    });
```

Comment:
Might be a nitpick, but since this test step indicates that the applicant has no DTC, we should include the Playwright actions (`check()`,`click()`) that indicate that the applicant has no DTC. These actions are currently in the previous test step.